src/communication/network_interface_sr.c

```diff
@@ -7577,6 +7612,9 @@ smmon_get_tran_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request, i
       /* 3) packing information
        *    - packing information with sequence of
        *      server info -> transaction info */
+      buffer = PTR_ALIGN (buffer_a, MAX_ALIGNMENT);
+      size -= buffer - buffer_a;
+
```

Comment:
To double check the size,
how about add an assert below at 7628L, when packing is done?

assert ( size == (ptr - buffer));

Reply:
I think that's enough.